cypress/integration/home.spec.ts

```diff
@@ -1,3 +1,37 @@
+it('should should display corresponding information', () => {
+  cy.visit('/')
+
+  cy.title().should(
+    'equal',
+    'Homepage | Octoclairvoyant: Compare GitHub changelogs across multiple releases'
+  )
+})
+
+it('should display h1 heading with correct text', () => {
+  cy.visit('/')
+
+  cy.findByRole('heading', { level: 1, name: 'Octoclairvoyant' }).should(
+    'exist'
+  )
+
+  cy.findByRole('heading', {
+    level: 2,
+    name: 'Compare GitHub changelogs across multiple releases',
+  }).should('exist')
+
+  // TODO: check 'src' property of the found img tag is correct
+  // NOTE: Testing playground suggests: cy.findByRole('img', { name: /octoclairvoyant reading a crystal ball/i })
+  cy.findByAltText(/octoclairvoyant reading a crystal ball/i).should('exist')
```

Comment:
Not sure how to drill down to get the `src` property out of the found element....